docker-compose.yml

```diff
@@ -4,8 +4,8 @@ services:
   postgres_development:
     image: postgres:13.1
     environment:
-      - POSTGRES_USER=user
-      - POSTGRES_PASSWORD=password
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=
```

Comment:
what's the reason for renaming this? didn't it work with the original env variables?